project/Dependencies.scala

```diff
@@ -1,19 +1,22 @@
-import sbt._
+import sbt.*
 
 object Dependencies {
 
-  val `config-tools`       = "com.evolutiongaming"        %% "config-tools"       % "1.0.4"
-  val `future-helper`      = "com.evolutiongaming"        %% "future-helper"      % "1.0.6"
-  val sequentially         = "com.evolutiongaming"        %% "sequentially"       % "1.1.4"
-  val `akka-serialization` = "com.evolutiongaming"        %% "akka-serialization" % "1.0.4"
-  val nel                  = "com.evolutiongaming"        %% "nel"                % "1.3.4"
-  val `safe-actor`         = "com.evolutiongaming"        %% "safe-actor"         % "3.0.0"
-  val `scala-tools`        = "com.evolutiongaming"        %% "scala-tools"        % "3.0.5"
-  val scalatest            = "org.scalatest"              %% "scalatest"          % "3.2.14"
+  val `config-tools`       = "com.evolutiongaming"        %% "config-tools"       % "1.0.5"
+  val `future-helper`      = "com.evolutiongaming"        %% "future-helper"      % "1.0.7"
+  val sequentially         = "com.evolutiongaming"        %% "sequentially"       % "1.2.0"
```

Comment:
1.2.0 and 1.1.5 are source and binary compatible. Since MiMa was introduced only in 1.2.0, I'm not sure whether it extends to 1.1.4 but I believe it should be as the only change for 1.1.5 was Scala 3 introduction.